ci/codecov.yml

```diff
@@ -10,19 +10,21 @@ coverage:
   status:
     patch:
       default:
-        target: '80'
+        target: 0
         if_no_uploads: error
         if_not_found: success
         if_ci_failed: failure
     project:
       default: false
       library:
-        target: auto
+        target: 0
         if_no_uploads: error
         if_not_found: success
         if_ci_failed: failure
         paths: '!lib/.*/tests/.*'
-
       tests:
-        target: 97.9%
```

Comment:
The 97.9 was a hand tuned value from looking at the coverage on the tests and being happy that what was missed was ok to miss.